asttokens/mark_tokens.py

```diff
@@ -162,28 +174,38 @@ def handle_comp(self, open_brace, node, first_token, last_token):
   # https://bugs.python.org/issue31241
   if sys.version_info < (3, 8):
     def visit_listcomp(self, node, first_token, last_token):
+      # type: (NodeNG, util.Token, util.Token) -> Tuple[util.Token, util.Token]
       return self.handle_comp('[', node, first_token, last_token)
 
   if six.PY2:
     # We shouldn't do this on PY3 because its SetComp/DictComp already have a correct start.
     def visit_setcomp(self, node, first_token, last_token):
+      # type: (NodeNG, util.Token, util.Token) -> Tuple[util.Token, util.Token]
       return self.handle_comp('{', node, first_token, last_token)
 
     def visit_dictcomp(self, node, first_token, last_token):
+      # type: (NodeNG, util.Token, util.Token) -> Tuple[util.Token, util.Token]
       return self.handle_comp('{', node, first_token, last_token)
 
-  def visit_comprehension(self, node, first_token, last_token):
+  def visit_comprehension(self,
+                          node,  # type: util.Token
```

Comment:
```suggestion
                          node,  # type: NodeNG
```

Is it possible to make a type alias to apply to all these methods?

Reply:
Maybe. Not sure how!